zerver/lib/markdown/fenced_code.py

```diff
@@ -194,27 +194,30 @@ def generic_handler(
     elif lang == "spoiler":
         return SpoilerHandler(processor, output, fence, header)
     elif lang in (
-        "blockdiag",
-        "seqdiag",
-        "nwdiag",
-        "actdiag",
-        "mermaid",
-        "plantuml",
-        "svgbob",
-        "packetdiag",
-        "rackdiag",
-        "graphviz",
-        "erd",
-        "excalidraw",
-        "vega",
-        "vegalite",
-        "ditaa",
-        "nomnoml",
-        "bpmn",
-        "bytefield",
-        "wavedrom",
-        "c4plantuml",
-        "umlet",
+        "kroki" + s
```

Comment:
Did we want a delimiter like a space here?

Reply:
Yea we want a space delimiter here, I'll add it